### DISCUSSION TITLE
ask for help about YOLOV5

### DISCUSSION THREAD

**Discussion by devil1213** [obss/sahi#661] [Q&A] [answered]
How to add SAHI to the source code of the YOLOv5 model that has been trained

**Comment by fcakyon** [comment]
@devil1213 can you be more specific on your question please?

**Comment by devil1213** [comment]
> @devil1213 can you be more specific on your question please?

I downloaded the yolov5 source code on github and trained the model. How can I add your proposed SAHI to the YOLv5 source code! thank you for your reply!

**Comment by fcakyon** [comment, marked as answer]
@devil1213 after training your yolov5 model, follow this tutorial to perform `sahi+yolov5` inference: https://github.com/obss/sahi/blob/main/demo/inference_for_yolov5.ipynb
